docs/src/routes/components/components/forms/form-notification/test/+page.md

```diff
@@ -230,7 +230,7 @@ Het basisvoorbeeld met een invoerveld en een verzendknop.
       aria-invalid="true"
     />
     <p class="error" id="input-link-error-message">
-      <span>Foutmelding:</span> Lorem <a href="form-notification-test">ipsum dolor</a> set amet
+      <span>Foutmelding:</span> Lorem <a href="/">ipsum dolor</a> set amet
```

Comment:
Generated links in markdown are rebased onto `{base}` (by [A.svelte#L6](https://github.com/minvws/nl-rdo-manon/blob/d9188e1/docs/src/markdown/components/A.svelte#L6) via the [mdsvex layout](https://github.com/minvws/nl-rdo-manon/blob/d9188e1/docs/src/markdown/layouts/default.svelte#L3)). However, this doesn't affect HTML content from the markdown source, which is a problem we ran into before with the `<iframe>`s. We now have a [remark plugin to fix that](https://github.com/minvws/nl-rdo-manon/blob/d9188e/docs/src/markdown/remark/iframe.js) for iframes. Do we want the same for HTML `<a>`s in markdown?